### ISSUE TITLE
`--print-timing-info` does not print timing info when given a json file graph input

### ISSUE THREAD

**Issue by researcherben** [sstsimulator/sst-core#1012]
`--print-timing-info` does not print timing info when given a json file graph input

Reproducer: replace `foobar.py` with a working architecture python file, this should act as a reproducer.

```bash
sst --output-json foobar.json foobar.py
sst --print-timing-info foobar.json
```

This should print the desired result.
```bash
sst --print-timing-info foobar.py
```

**Comment by feldergast** [comment]
That is odd behavior since the print-timing-info command line option and the model definition used to build the graph are completely independent code (or should be).  I'll take a look.

**Comment by feldergast** [comment]
Command line arguments should override what’s in the input file.  However, if the option only allows a one-way flag (i.e. I can only turn something on and not off), then there would be no way to turn something off on the command line that was turned on in the input file.  This may mean we need to make all flags take an optional argument.  Not sure if this is true today, but changing that would be pretty easy.

**Comment by feldergast** [comment]
I just checked and print-timing-info can take an argument, so setting --print-timing-info=false should override what's set in the input file.  Once I have a chance to test that this works as expected I'll close the issue.

**Comment by feldergast** [comment]
Looks like the command line override of options set in the input file is broken for all options.  I'll put in a fix and a test to make sure it doesn't break again.

**Comment by feldergast** [comment]
Fixed in #1017
